src/wallet/rpcdump.cpp

```diff
@@ -917,6 +918,46 @@ static UniValue ProcessImport(CWallet * const pwallet, const UniValue& data, con
                 pwallet->SetAddressBook(dest, label, "receive");
             }
 
+            // Import public keys
+            for (size_t i = 0; i < pubKeys.size(); ++i) {
+                CPubKey pubKey = HexToPubKey(pubKeys[i].get_str());
+
+                if (!pubKey.IsFullyValid()) {
+                    throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, "Pubkey is not a valid public key");
+                }
+
+                CTxDestination pubkey_dest = pubKey.GetID();
```

Comment:
nit: `pubkey_dest` => `pubKeyDest`?

Reply:
No. The style guide says to use snake case for variable names.

Reply:
Any reason why the other local variables don't follow this naming convention? (For example, `pubKey` and `pubKeyScript`)

Reply:
The style convention has changed over time. New code should follow the new convention. Changing all of the existing code to follow that would break a ton of PRs so we don't do that.

Reply:
Gotcha. So the variables I named are `new code` AFAICT - though I may be missing something. They appear to be new variables defined as part of the work done for this PR.